src/components/modules/VideoCard/index.tsx

```diff
@@ -1,20 +1,27 @@
 import React from "react";
+import { LinkTo } from "components/atoms";
 import * as S from "./style";
 
 type VideoProps = {
```

Comment:
VideoList에서 재사용된다면 export 해서 사용해도 될거같아요!

```suggestion
export type VideoProps = {
```